docs/en/query_language/agg_functions/parametric_functions.md

```diff
@@ -276,17 +276,137 @@ ORDER BY level
 
 Simply, the level value could only be 0, 1, 2, 3, it means the maxium event action stage that one user could reach.
 
-## retention(cond1, cond2, ...)
+## retention {#retention}
 
-Retention refers to the ability of a company or product to retain its customers over some specified periods.
+Analytical function that shows how much 
```

Comment:
* Looks incomplete or messed up
* "Analytical function" is sometimes used with different meaning, so I'd avoid that